File: TUF/Array/AverageOfAllElm.java
import java.util.Scanner;

public class AverageOfAllElm {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the size of the array: ");
        int size = input.nextInt();
        int[] arr = new int[size];
        for (int i = 0; i < size; i++) {
            System.out.printf("Enter %d number element of the array: ", i + 1);
            arr[i] = input.nextInt();
        }
        input.close();
        System.out.println("Average of all elements: " + findAvgOfAllElm(arr));
    }

    private static double findAvgOfAllElm(int[] arr) {
        double sum = 0;
        for (int i = 0; i < arr.length; i++) {
            sum += (double) arr[i];
        }
        return sum / arr.length;
    }
}

File: TUF/Array/AverageOfAllElm.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
function findAvgOfAllElm(arr) {
    var sum = 0;
    for (var _i = 0, arr_1 = arr; _i < arr_1.length; _i++) {
        var i = arr_1[_i];
        sum += i;
    }
    return sum / arr.length;
}
var arr = [1, 2, 1, 1, 5, 1];
console.log("Average of all elements: ".concat(findAvgOfAllElm(arr)));
